util/crontab-example

```diff
@@ -0,0 +1,2 @@
+SHELL=/bin/bash
+0 7 * * * cd $HOME/wptdashboard && ./run/run.py firefox-56.0-linux --upload --create-testrun &>> /var/log/wptd.log
```

Comment:
This would be good to add as a comment above as well I think :-) `#` should work for comments in crontab files I think.